### ISSUE TITLE
FE Remove "Other nearby VA locations" from the bottom of Lovell  TRICARE Locations page

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#11829]
Remove "Other nearby VA locations" from the bottom of the locations page for Lovell Federal TRICARE health care.

https://web-liqqgmzl7tsyfbwyulbkuborufuuttav.demo.cms.va.gov/lovell-federal-tricare-health-care/locations/
![Screenshot 2022-12-06 at 3 27 48 PM](https://user-images.githubusercontent.com/21045418/206015860-253dd580-2757-407a-bf16-ab004828eb91.png)

The "Other nearby VA locations" section does not apply to Lovell TRICARE because any nearby locations it can find in the facility API will NOT be TRICARE so any that show would be misleading.

## ACs
- [ ] "Other nearby VA locations" should not appear on /lovell-federal-tricare-health-care/locations/  (Specific to Lovell TRICARE)
- [ ] Bonus for making it disappear whenever the widget gets no results  (should not see the heading if the widget is empty.)  (Generic to all instances)

**Comment by JQuaggles** [comment]
Is it possible (or make sense) to do a Nearby Tricare locations or add in a link that takes you to a Tricare MTF page? Or is the nearby locations a toggle for the page as opposed to customizing? @davidmpickett 
   
   Header: Other nearby Tricare locations
   Text link: For other nearby Tricare locations, head to the TRICARE - Find a Military Hospital"
   
Or something - that being said, the next closest MTF/Clinic to Lovell is about 150 Mi away, so that might not be the best idea.

**Comment by davidmpickett** [comment]
@JQuaggles My understanding is that on the VA side The Nearby Locations module is hardcoded based on information pulled from VAST or Facilities API. The TRICARE facilities are excluded from this data source so this will section will not return any results. Hiding this section on TRICARE is the path of least resistance to launch.

**Comment by JayDarnell** [comment]
@davidmpickett @JQuaggles - @swirtSJW and I jammed on this issue for a bit yesterday so I could better understand the cause of this problem. As it turns out, the issue with Lovell as outlined in this ticket is a content problem. The Lovell TRICARE Locations list page incorrectly had all the tricare facilities listed as Other Facilities. This is a problem for a few reasons:

1. This is redundant - the other facilities field is only needed to include facilities outside of the current system
2. This won't work for TRICARE facilities because the connection is made based on the Facility Locator API ID and this value for tricare facilities ONLY exists within the CMS. The way this section is populated is by a front end widget that looks for a div with a specific attribute, checks a second attribute on the div containing a list of facility api ids, looks up those facilities on the front end (similar to the facility locator itself - full disclosure I did not go so far down the rabbit hole as to dig into this code) and inserts them into the div
3. Because the TRICARE facility locator api ids aren't in VAST or lighthouse it couldn't find them to display them (and they shouldn't be in VAST or lighthouse anyway
4. This problem can happen outside of Lovell too. If an editor for a given system manually adds a facility locator api id for a facility into the Other Facilities field and that facility moves systems or undergoes any changes in VAST that would change its facility locator api id (and subsequently remove the original record) then the section would still display but be blank because the widget can no longer find the item it needs to display

This call all be fixed by reworking this section to pull the data from the CMS instead of the front end and display it using liquid templates like we do in other areas of the sites. BUT it can only do this once the CMS is the source of truth for all facilities.
